src/autocomplete.js

```diff
@@ -312,6 +318,7 @@
             return data.filter(x => ~x[this.options.nameProperty].toLowerCase().indexOf(input.toLowerCase()));
         },
         filterOn: 'input',
+        onFocus: 'click',

```

Comment:
I would propose to rename this property to "openOnInput" which takes a boolean.

Reply:
Would you like to have the possibility to set this with HTML attributes: e.g. ignore-open-on-input?

Reply:
Sure why not. It's not a requirement, but nice to have. Though I think `open-on-input` would be a better name for the attribute (same as the property).